### PR TITLE
Feat/organization card

### PR DESCRIPTION
**Pull request by amphikapha**
### Type of Change

- [x] New features
- [x] Bug fixes
- [ ] Breaking changes
- [ ] Refactor
- [ ] Documentation updates
- [ ] Other (please specify):

## Description

An organizationCard fits for all screen size.
I also add hover transition for the right arrow of desktop screen. 

## Screenshots or GIFs (if applicable)

image

image

image

### CODE REVIEW

File: src/lib/components/OrganizationCard/OrganizationCard.svelte [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+							'leading-6 whitespace-pre-line text-wrap absolute bottom-0 pr-8',
+							'max-2xl:text-xl max-xl:text-xl max-lg:text-base max-lg:leading-5'
+						)}
+					>
+						{organization.orgName}
+					</div>
+					<div
+						class="h-[32px] w-[32px] absolute bottom-0 right-0 flex items-center justify-center transform transition-transform duration-300 group-hover:translate-x-1"
+					>
+						<img src={arrowRight} alt="arrowRight" width="24" height="auto" />
```
Comment (punchanabu) [inline src/lib/components/OrganizationCard/OrganizationCard.svelte]: This one you can use `svelte-fa` and `font-awesome` instead

```typescript
import Fa from 'svelte-fa';
import { faArrowRight } from '@fortawesome/free-solid-svg-icons';

<Fa icon class={faArrowRight} />
```

File: src/lib/components/OrganizationCard/OrganizationCard.svelte [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+			imageURL: sgcu,
+			orgName: 'องค์การบริหารสโมสรนิสิต จุฬาลงกรณ์มหาวิทยาลัย (อบจ.)',
+			linkHref: 'https://www.chula.ac.th/academics/academic-units/sgcu/'
+		},
+		{
+			imageURL: sapanisit,
+			orgName: 'สภานิสิต \n จุฬาลงกรณ์มหาวิทยาลัย',
+			linkHref:
+				'https://www.sa.chula.ac.th/%E0%B8%AA%E0%B8%A0%E0%B8%B2%E0%B8%99%E0%B8%B4%E0%B8%AA%E0%B8%B4%E0%B8%95/'
+		}
```
Comment (punchanabu) [inline src/lib/components/OrganizationCard/OrganizationCard.svelte]: can you move this part of code into `src/lib/constant` and add a file `organizations.ts` and import this variables to this part of code instead ?

File: src/lib/components/OrganizationCard/OrganizationCard.svelte [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+			<div
+				class={cn(
+					typography({ variant: 'body-medium' }),
+					'h-[72px] w-[202px] leading-5 font-semibold whitespace-pre-line text-wrap flex items-center'
+				)}
+			>
+				{organization.orgName}
+			</div>
+			<div class="w-[35px] h-[35px] flex-shrink-0 ml-auto mt-auto">
+				<img src={arrowCircleRight} alt="arrowCircleRight" width="35" height="35" />
```
Comment (punchanabu) [inline src/lib/components/OrganizationCard/OrganizationCard.svelte]: This one too change to `svelte-fa` and `fontawesome`. I think the icons name is `faArrowCircleRight`

File: src/lib/assets/icons/arrow-circle-right.svg
Comment (punchanabu) [inline src/lib/assets/icons/arrow-circle-right.svg:1]: so if we use `font-awesome` you can basically just remove this

File: src/lib/assets/icons/arrow-right.svg
Comment (punchanabu) [inline src/lib/assets/icons/arrow-right.svg:1]: we can remove this too since we are using `font-awesome`

**Review by punchanabu — CHANGES_REQUESTED** [review]

File: src/lib/components/OrganizationCard/OrganizationCard.svelte (hunk 3 above)
Comment (amphikapha) [reply src/lib/components/OrganizationCard/OrganizationCard.svelte]: this icon is not the same as in figma na, but it's not that bad na

image

File: src/lib/components/OrganizationCard/OrganizationCard.svelte [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+						class={cn(
+							typography({ variant: 'heading4' }),
+							'leading-6 whitespace-pre-line text-wrap absolute bottom-0 pr-8',
+							'max-2xl:text-xl max-xl:text-xl max-lg:text-base max-lg:leading-5'
+						)}
+					>
+						{organization.orgName}
+					</div>
+					<div
+						class="h-[32px] w-[32px] absolute bottom-0 right-0 flex items-center justify-center transform transition-transform duration-300 group-hover:translate-x-1"
```
Comment (Chulinuwu) [inline src/lib/components/OrganizationCard/OrganizationCard.svelte]: tailwind unit -- <div
						class="h-8 w-8 absolute bottom-0 right-0 flex items-center justify-center transform transition-transform duration-300 group-hover:translate-x-1"

File: src/lib/components/OrganizationCard/OrganizationCard.svelte (hunk 3 above)
Comment (amphikapha) [reply src/lib/components/OrganizationCard/OrganizationCard.svelte]: fixed by punpun. thank you teacher

image

**Review by punchanabu — APPROVED** [review]
Good Job!

**Comment by punchanabu** [comment]
ฝากลบ svg ไอค่อนเก่าออก

**Comment by punchanabu** [comment]
แล้ว merge เลย

**Pull request merged**